model/src/main/proto/javaclasses/mealorder/model.proto

```diff
@@ -33,60 +33,69 @@ option java_generate_equals_and_hash = true;
 import "javaclasses/mealorder/identifiers.proto";
 import "javaclasses/mealorder/values.proto";
 
+// The model representing a dish.
+//
+// This is one item from the menu, represented by the name of the dish, its price and categoty.
+//
 message Dish {
 
+    // An identifier of the dish.
     DishId id = 1;
 
+    // Name of the dish.
     string name = 2;
 
+    // A category to which the dish belongs.
     string category = 3;
 
-    //TODO: How to use BigInteger in protobuf?
+    // A dish price.
     int64 price = 4;
```

Comment:
Why not have a value for `Money`? It would eliminate questions like: “In which currency is this price? Is this minor currency units or major?”